src/compiler/sscp/TargetSeparationPass.cpp

```diff
@@ -284,6 +302,8 @@ std::unique_ptr<llvm::Module> generateDeviceIR(llvm::Module &M,
     }
   }
 
+  replaceInvalidCharsInSymbolNames(*DeviceModule);
```

Comment:
Please only do this on Windows. Otherwise all our symbols may get non-demangleable.

Reply:
Well, the implementation already only did it on windows, felt like it was less annoying to not have macros here, but macrod out all of it.